rocky/katalogus/templates/partials/plugin_tile.html

```diff
@@ -1,7 +1,9 @@
 {% load i18n %}
 {% load static %}
 
-<div role="group" id="plugin_{{ plugin.id|slugify }}">
+<div role="group"
+     class="plugin {% if plugin.enabled %} plugin-is-enabled {% else %} plugin-is-disabled {% endif %}{% if hidden %} overflow-item {% endif %}"
```

Comment:
I think we can do better than the overflow-item class.
Could we toggle a class on the grouping div that shows/hides overflowed items in general, and then use the css 
```
.overflow-too-many-hidden div.nth-of-class('.plugin'){
  display:none;
}
```
That way you'd only have to toggle(.overflow-to-many-hidden) on the grouping parent.
This could also be  more generic than plugins only,